skops/io/_dispatch.py

```diff
@@ -110,15 +110,17 @@ def _get_trusted(trusted, default):
 
         This is a convenience method called by child classes.
         """
+        _defaults = cast_defaults_to_strings(default)
```

Comment:
Technically, this can be moved below the first return (`if trusted is True...`), resulting in a minuscule performance increase.